source/blacklist.rst

```diff
@@ -136,11 +137,11 @@ Switching the blacklist to whitelist mode (via the :doc:`../config`) will invert
 Examples
 ========
 
-.. topic:: Example: Blocking all bucket use
+.. topic:: Example: Blocking some bucket use
```

Comment:
Turns out there are way too many different bucket materials now, but I couldn't think of a better example so I just reworded this^^